build.gradle

```diff
@@ -57,6 +62,12 @@ task chopManualRu(dependsOn: [buildManualRu], type: CreateMultiPageDoc) {
     docTitle = "<a href='https://www.cuba-platform.ru/manual' target='_blank'>Платформа CUBA</a>. Руководство по разработке приложений"
 }
 
+task chopManualCn(dependsOn: [buildManualEn], type: CreateMultiPageDoc) {
```

Comment:
task `chopManualCn` should depend on `buildManualCn`, not En.

Reply:
Hi,thanks.I have fixed that.